docs/pages/router/assets/fonts.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Fonts
+---
+
+Expo Router's `SplashScreen` can be used to hide your application until your desired font's are loaded.
+
+```tsx file=app/_layout.tsx
```

Comment:
```suggestion
```tsx app/_layout.tsx
```

This suggestion applies to most of the added CodeBlock titles, probably using Regex to correct that would be the best approach.

Reply:
I'm working on formatting all the pages 👍